GameData/RP-1/Contracts/Earth Observation 1/EOSIRRad1.cfg

```diff
@@ -0,0 +1,128 @@
+
+	synopsis = Launch the first weather satellite with the Infrared Radiometry experiment and an Early TV camera.
+
+	completedMessage = Success! The images provided from this satellite have proven that weather satellites can be extremely effective in helping meteorologists predict the weather.
+
+	sortKey = 402
+
+	cancellable = true
+	declinable = false
+	autoAccept = false
+	minExpiry = 0
+	maxExpiry = 0
+	maxCompletions = 1
+	maxSimultaneous = 1
+	deadline = 0
+
+	targetBody = HomeWorld()
+
+	// ************ REWARDS ************
+	prestige = Trivial       // 1.0x
+	advanceFunds = 0
+	rewardScience = 0
+	rewardFunds = 0
+	failureFunds = 0
+	rewardReputation = 60
+	failureReputation = 0 // was @rewardReputation
+
+	// ************ REQUIREMENTS ************
+
+	REQUIREMENT
+	{
+		name = ProgramActive
+		type = ProgramActive
+		program = EarlyEOS
+	}
+
+
+	PARAMETER
+	{
+		name = ScienceSatIRRad
+		type = VesselParameterGroup
+		title = First weather satellite
+		define = ScienceSatIRRad
+
+		PARAMETER
+		{
+			name = NewVessel
+			type = NewVessel
+			title = Launch a new vessel
+			hideChildren = true
+		}
+		PARAMETER 
+		{
+			name = Crewmembers
+			type = HasCrew
+			minCrew = 0
+			maxCrew = 0
+			title = Uncrewed
+			hideChildren = true
+		}
+
+		PARAMETER
+		{
+			name = OrbitSequence
+			type = Sequence
+			title = Survive in orbit for a day and transmit science
+
+			PARAMETER
+			{
+				name = HasPower
+				type = HasResource
+				resource = ElectricCharge
+				minQuantity = 1.0
+				title = Craft must have ElectricCharge after one day
+				hideChildren = true
+			}
+
+			PARAMETER
+			{
+				name = Orbit
+				type = Orbit
+				maxApA = 1000000
+				minPeA = 500000
+				maxEccentricity = 0.005
+				disableOnStateChange = true
+				title = Achieve orbit between 500km and 1,000km in altitude with no more than .005 eccentricity.
+			}
+
+			PARAMETER
+			{
+				name = Duration
+				type = Duration
+				duration = 1d
+				preWaitText = Check for stable orbit
+				waitingText = Checking for stable orbit
+				completionText = Stable orbit: Confirmed
+			}
+		
+			PARAMETER
+			{
+				name = RP1CollectScience
+				type = RP1CollectScience
+				targetBody = HomeWorld()
+				situation = InSpaceLow
+				experiment = RP0infraredRad1
+				title = Transmit infrared weather data from low orbit
+			}
+			PARAMETER
+			{
+				name = RP1CollectScience
+				type = RP1CollectScience
+				targetBody = HomeWorld()
+				situation = InSpaceLow
+				experiment = RP0visibleImaging1
+				fractionComplete = 1

```

Comment:
I don't think you want 1 here, especially for a non-polar contract. This is biome-dependent so 1 would mean "complete data from all biomes", including (a) poles, and (b) shores, which are super tough to get at warp. You probably want something pretty low here and to more rely on the IR one.